herddb-core/src/main/java/herddb/server/ServerConfiguration.java

```diff
@@ -245,11 +245,10 @@
     public static final String PROPERTY_JMX_ENABLE = "server.jmx.enable";
     public static final boolean PROPERTY_JMX_ENABLE_DEFAULT = true;
 
-    public static final String PROPERTY_PLANNER_TYPE = "server.planner.type";
-    public static final String PLANNER_TYPE_JSQLPARSER = "jsqlparser";
```

Comment:
This option is not working anymore since many major versions, it worked only during the switch to Calcite